x-pack/plugins/osquery/public/saved_queries/saved_query_flyout.tsx

```diff
@@ -48,7 +48,14 @@ const SavedQueryFlyoutComponent: React.FC<AddQueryFlyoutProps> = ({ defaultValue
 
   return (
     <EuiPortal>
-      <EuiFlyout size="m" ownFocus onClose={onClose} aria-labelledby="flyoutTitle">
+      <EuiFlyout
+        size="m"
+        ownFocus
+        onClose={onClose}
+        aria-labelledby="flyoutTitle"
+        // eslint-disable-next-line react-perf/jsx-no-new-object-as-prop
+        maskProps={{ style: 'z-index: 6000' }} // For an edge case to display above the alerts flyout
```

Comment:
So I see below that this flyout has a form in it, did you verify that all the select dropdowns etc that render in this form, namely EuiSuperSelect options and anything that renders itself via a portal (not sure of everything) render correctly? I noticed recently that anything that renders itself in a portal and it's parent is also in a portal, that then tries to determine it's own stacking context cannot have a z-index higher than 2000 or something. https://codesandbox.io/s/columntypesgrid-material-demo-forked-b7hpb4?file=/demo.js for a demo. Mentioned this to eui but they didn't seem to think it's a bug, personally I do, https://github.com/elastic/eui/blob/main/src/services/popover/popover_positioning.ts#L743 for an element in a portal will not do what it should.

Reply:
Hey, I don't know yet, but will take a look at this and get back to you. Big thanks! :)

Reply:
Thanks, I double checked and it seems to be working just fine. Selects too :) 
The only thing is the Global NavBar that doesnt really work when any of the additional flyouts is opened.

Reply:
So I checked another time, and apparently it works fine on Alerts, but messes up one of the flyouts in Osquery itself. 

Thanks for pointing this out :)

Reply:
the issue is fixed here https://github.com/elastic/kibana/pull/130944